platform/linux-generic/odp_queue_scalable.c

```diff
@@ -246,7 +241,7 @@ static int queue_init_global(void)
 		queue = get_qentry(i);
 		LOCK_INIT(&queue->s.lock);
 		queue->s.index  = i;
-		queue->s.handle = queue_from_id(i);
+		queue->s.handle = (odp_queue_t)queue;
```

Comment:
`queue_to_ext()` here?

Reply:
I think it's better to show explicitly that handle is qentry pointer. Actually, this qentry->s.handle is not needed as it's value is qentry itself. But didn't remove it now to limit the number of changes (and it's easy to change back if needed).